plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/connection/DBPDriverLibrary.java

```diff
@@ -98,6 +98,11 @@ public static FileType getFileTypeByFileName(String fileName) {
     @Nullable
     Path getLocalFile();
 
+    /**
+     * Returns CRC of library file.
+     */
+    long getFileCRC();

```

Comment:
CRC32 is 32-bit. Can we please update `DriverDescriptor#calculateFileCRC` so it returns `int`, please? Pretty please?

Reply:
I'm 99% sure this API was designed around the current implementation of `DriverDescriptor#calculateFileCRC`. CRC is the name of the family of algorithms. If the API is supposed to be algorithm-agnostic, it should've been called `getFileChecksum` instead, not `getFileCRC`.